packages/elements-dev-portal/src/hooks/useGetWorkspace.tsx

```diff
@@ -2,11 +2,12 @@ import * as React from 'react';
 import { useQuery } from 'react-query';
 
 import { PlatformContext } from '../components/DevPortalProvider';
+import { devPortalCacheKeys } from '../consts';
 import { getWorkspace } from '../handlers/getWorkspace';
 
 export function useGetWorkspace({ projectIds }: { projectIds: string[] }) {
   const { platformUrl, platformAuthToken } = React.useContext(PlatformContext);
-  return useQuery(['useWorkspace', projectIds, platformUrl, platformAuthToken], () =>
+  return useQuery([...devPortalCacheKeys.searchNodes({ projectIds }), platformUrl, platformAuthToken], () =>
```

Comment:
Shouldn't it have a separate key for the workspace?

Reply:
it's basically a search